### ISSUE TITLE
Some question of KLD

### ISSUE THREAD

**Issue by alanlisten** [zfjsail/gae-pytorch#3]
`KLD = -0.5 / n_nodes * torch.mean(torch.sum(1 + 2 * logvar - mu.pow(2) - logvar.exp().pow(2), 1))`
/ n_nodes  should be removed or
torch.mean → torch.sum

**Comment by AllenWu18** [comment]
then is it should be 
KLD  = -0.5 * torch.mean(torch.sum(1 + 2 * logvar - mu.pow(2) - logvar.exp().pow(2), 1))
or 
KLD = -0.5 / n_nodes * torch.sum(torch.sum(1 +2 * logvar - mu.pow(2)-logvar.exp().pow(2) , 1))?

**Comment by hylBen** [comment]
@alanlisten @AllenWu18 you both are right about the KLD. The author has admitted that 1 / n_nodes serves as a rescaling parameter (like \beta in beta-vae) to weaken the regularization from KLD. Check this issue: [Loss function in optimizer.py #20](https://github.com/tkipf/gae/issues/20).

**Comment by Dzhilin** [comment]
Please tell me what is the meaning of "norm" in the loss function? Looking forward to your reply！
